src/pages/homepage/HomePage.jsx

```diff
@@ -40,6 +42,7 @@ const Trending = lazy(() =>
 );
 
 // Import scoped SCSS
+import Modal from '@/components/cart/Modal';
```

Comment:
Not clear from component name that this is cart, how about naming the component something more specific like CartModal?

Reply:
✅

Reply:
https://github.com/algolia/algolia-react-boilerplate/pull/436#:~:text=now%20in%20recoil-,fix%3A%20rename%20modal%20in%20Cart%20Modal,-c69041e